Quotient/events/roommessageevent.cpp

```diff
@@ -6,7 +6,9 @@
 #include "roommessageevent.h"
 
 #include "../logging_categories_p.h"
+#include <Quotient/converters.h>
 #include "eventrelation.h"
+#include <Quotient/events/event.h>
```

Comment:
```suggestion
#include "eventrelation.h"
```
These two are already included via `roommessageevent.h`.